code/game/objects/items/stacks/medical.dm

```diff
@@ -285,6 +284,55 @@
 		return
 	. = ..()
 
+/obj/item/stack/medical/mesh/advanced
+	name = "advanced regenerative mesh"
+	desc = "An advanced mesh made with aloe extracts and sterilizing chemicals, used to threat burns."
+	gender = PLURAL
+	singular_name = "advanced regenerative mesh"
+	icon_state = "aloe_mesh"
+	heal_burn = 15
+	grind_results = list(/datum/reagent/medicine/spaceacillin = 1)
+
+/obj/item/stack/medical/mesh/advanced/update_icon_state()
+	if(!is_open)
+		icon_state = "aloe_mesh_closed"
+	else
+		return ..()
+
+/obj/item/stack/medical/aloe
+	name = "aloe cream"
+	desc = "An healing paste you can apply on wounds."
+	icon_state = "aloe_paste"
+	self_delay = 20
+	other_delay = 10
+	novariants = TRUE
+	amount = 20
+	max_amount = 20
+	var/heal = 3
+	grind_results = list(/datum/reagent/consumable/aloejuice = 1)
+
+/obj/item/stack/medical/aloe/heal(mob/living/M, mob/user)

```

Comment:
should you have to do this?

Reply:
the parent doesnt have heals and a ssubtype of subtype is gonna rewrite it anyway